<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BackupMonitorManageGroupsEditor</class>
 <widget class="QWidget" name="BackupMonitorManageGroupsEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Name:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="inputName"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Beschreibung:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QPlainTextEdit" name="inputDescription"/>
   </item>
   <item row="2" column="1">
    <widget class="QCheckBox" name="checkBoxAutoExpand">
     <property name="text">
      <string>Automatisch aufklappen</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
